rpc/Engine.postman_collection.json

```diff
@@ -1986,183 +1993,728 @@
-								"header": [
-									{
-										"key": "Content-Type",
-										"name": "Content-Type",
-										"value": "application/json",
-										"type": "text"
-									}
-								],
-								"body": {
-									"mode": "raw",
-									"raw": "{\n\t\"method\": \"vaults.linked.get_linked_data\",\n\t\"params\": {\n\t\t\"storageCredentials\": \"{{storage_credential_id}}\",\n\t\t\"vaultWallet\": \"{{wallet_id}}\",\n\t\t\"linkEntry\": {\n\t\t\t\"remoteVault\": \"81a17f6d-1de6-459c-b34d-e725a63eaf21\",\n\t\t\t\"remoteWallet\": \"e059ac52-da27-46cb-a6d0-9df941a8fd36\",\n\t\t\t\"remoteStorage\": \"2d7cd752-5d80-434e-a8e5-e0cec0e89649\",\n\t\t\t\"container\": \"Product\"\n\t\t}\n\t},\n\t\"jsonrpc\": \"2.0\",\n\t\"id\": 0\n}"
-								},
-								"url": {
-									"raw": "{{engine_url}}",
-									"host": [
-										"{{engine_url}}"
-									]
-								}
-							},
-							"response": []
-						}
-					],
-					"protocolProfileBehavior": {},
-					"_postman_isSubFolder": true
-				},
-				{
-					"name": "shipchain",
+					"name": "load.1.2.0",
 					"item": [
 						{
-							"name": "procurement",
+							"name": "tx",
 							"item": [
 								{
-									"name": "fields",
-									"item": [
+									"name": "load.1.2.0.create_shipment_tx",
+									"event": [
 										{
-											"name": "procurement.fields.get",
-											"request": {
-												"method": "POST",
-												"header": [
-													{
-														"key": "Content-Type",
-														"name": "Content-Type",
-														"type": "text",
-														"value": "application/json"
-													}
+											"listen": "test",
+											"script": {
+												"id": "c97f5e47-03ac-4486-95ed-02d0dc84fdea",
+												"exec": [
+													""
 												],
-												"body": {
-													"mode": "raw",
-													"raw": "{\n\t\"method\": \"vaults.shipchain.procurement.fields.get\",\n\t\"params\": {\n\t\t\"storageCredentials\": \"{{storage_credential_id}}\",\n\t\t\"vaultWallet\": \"{{wallet_id}}\",\n\t\t\"vault\": \"{{vault_id}}\"\n\t},\n\t\"jsonrpc\": \"2.0\",\n\t\"id\": 0\n}"
-												},
-												"url": {
-													"raw": "{{engine_url}}",
-													"host": [
-														"{{engine_url}}"
-													]
-												}
-											},
-											"response": []
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "POST",
+										"header": [
+											{
+												"key": "Content-Type",
+												"value": "application/json"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": "{\n\t\"method\": \"load.1.2.0.create_shipment_tx\",\n\t\"params\": {\n\t\t\"shipmentUuid\": \"{{shipment_uuid}}\",\n\t\t\"senderWallet\": \"{{wallet_id}}\", \n\t\t\"fundingType\": 1, \n\t\t\"contractedAmount\": 10000000000000, \n\t\t\"carrierWallet\": \"{{wallet_id}}\"\n\t},\n\t\"jsonrpc\": \"2.0\",\n\t\"id\": 0\n}"
```

Comment:
Since we updated the number parameters to strings, make sure they are updated in the Postman Collection as well